.github/workflows/test-convene-web.yml

```diff
@@ -34,6 +34,13 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install Overmind
+      run: |
+        sudo apt-get install tmux
+        sudo mkdir ~/bin/
+        sudo wget -O ~/bin/overmind https://github.com/DarthSim/overmind/releases/download/v2.2.1/overmind-v2.2.1-linux-386.gz
```

Comment:
May need to `gunzip` the resulting file, since it's likely gzipped given the `.gz` file extension :)